java/trace/src/main/scala/org/typelevel/otel4s/java/trace/TracerImpl.scala

```diff
@@ -59,4 +62,15 @@ private[java] class TracerImpl[F[_]: Sync](
 
   def noopScope[A](fa: F[A]): F[A] =
     scope.noopScope(fa)
+
+  def joinOrContinue[A, C: TextMapGetter](carrier: C)(fa: F[A]): F[A] = {
+    val context = propagators.textMapPropagator.extract(Vault.empty, carrier)
```

Comment:
Should we pass the currently available `Vault` from the `Local[F, Vault]`? 
The propagation works either way.

Reply:
According to the OpenTelemetry implementation, we can pass `null` as a JContext, and the root context will be selected as a fallback. 

I guess passing an empty `Vault` is ok there.